src/catalog/src/builtin.rs

```diff
@@ -7173,6 +7205,16 @@ ON mz_internal.mz_webhook_sources (id)",
     is_retained_metrics_object: true,
 };
 
+pub static MZ_CATALOG_EXTRACTOR_CONNECTION: Lazy<BuiltinConnection> =
+    Lazy::new(|| BuiltinConnection {
+        name: "mz_catalog_extractor_connection",
+        schema: MZ_INTERNAL_SCHEMA,
+        oid: oid::CONNECTION_MZ_CATALOG_EXTRACTOR_CONNECTION_OID,
+        sql: "CREATE CONNECTION mz_internal.mz_catalog_extractor_connection
+TO AWS (ASSUME ROLE ARN = '')",
+        access: vec![ANALYTICS_USAGE, ANALYTICS_UPDATE],
```

Comment:
`UPDATE` is not valid privilege for connections. Only `USAGE` is.

https://materialize.com/docs/manage/access-control/rbac/#privileges